### PR TITLE
Add test for unexpected stderr or errors

### PR DESCRIPTION
**Pull request by tstromberg**
To prevent issues like stray warnings.

### CODE REVIEW

File: test/integration/no_error_shown_test.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	if NoneDriver() {
+		t.Skip("none driver always shows a warning")
+	}
+	MaybeParallel(t)
+
+	profile := UniqueProfileName("nospam")
+	ctx, cancel := context.WithTimeout(context.Background(), Minutes(25))
+	defer CleanupWithLogs(t, profile, cancel)
+
+	// TODO: make this multi-node once it's stable
```
Comment (medyagh) [inline test/integration/no_error_shown_test.go:39]: Make this Todo an issue and link to the issue

File: test/integration/no_error_shown_test.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import (
+	"context"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+// TestMultiNodeNoErrorShown asserts that there are no errors displayed
+func TestNoErrorShown(t *testing.T) {
```
Comment (medyagh) [inline test/integration/no_error_shown_test.go:29]: The comment doesn't match the name.
How about a better name.
TestCleanDispalyStart ?

**Comment by tstromberg** [comment]
Test seems to be doing the correct thing now. Will wait to see if I can get the error it reported sorted out.

**Pull request merged**